eprihlaska/forms.py

```diff
@@ -64,9 +64,9 @@ def process_formdata(self, valuelist):
 class MoreDetailPersonalDataForm(FlaskForm):
     birth_no = StringField(label=c.BIRTH_NO,
                            validators=[BirthNoValidator()])
-    date_of_birth = ePrihlaskaDateField(label=c.BIRTH_DATE,
-                                        format='%d.%m.%Y',
-                                        default=DEFAULT_DATE)
+    date_of_birth = StringField(label=c.BIRTH_DATE,
+                                validators=[DateValidator()],
+                                default='DD.MM.RRRR')
```

Comment:
@JakubNvk Co by sme tu presne potrebovali je placeholder, ale to zial asi `StringField` neznesie...

Asi by som to odporucal sem nedat.

Reply:
Trosku sa bojim, ze tak polovica pouzivatelov zada zly format (zly podla nasich standardov) a budu to musiet menit, ak to tam nenechame.